src/client/mod.rs

```diff
@@ -274,20 +278,61 @@ impl<'a> AsRef<FetchPartition<'a>> for FetchPartition<'a> {
 
 // --------------------------------------------------------------------
 
+/// This will be expanded in the future. See #51.
+#[derive(Debug, Clone)]
+pub struct SecurityConfig(pub Ssl);
```

Comment:
could we make the wrapped `Ssl` private? this would make it possible for future changes to the `SecurityConfig` struct's internals not affect the public API.